src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -396,10 +396,7 @@ public ActiveDirectoryDomain getDomain(String domain) {
      *         <br>
      *         false if either the application is not in FIPS mode or any of requireTls, startTls is true.
      */
-    private static boolean isFipsNonCompliant(Boolean requireTls, Boolean startTls) {
-        // requireTls or startTls can be null, so in order to avoid null pointer exception assuming the values as FALSE.
-        requireTls = requireTls == null ? false : requireTls;
```

Comment:
This was giving spot bugs due to auto boxing and unboxing so reverted the change.